.github/workflows/diff.yml

```diff
@@ -6,25 +6,53 @@ jobs:
   render-diff:
     runs-on: ubuntu-latest
     steps:
-
       - name: Wait on Workflow
         uses: lucasssvaz/wait-on-workflow@v1
         with:
           workflow: pr.yml
           max-wait: 3
           timeout: 60
           sha: ${{ github.event.pull_request.head.sha || github.sha }}
+      
+      - name: Get PR description
+        id: get_pr
+        run: |
+          pr_number=$(jq --raw-output .number "$GITHUB_EVENT_PATH")
+          pr_description=$(gh pr view $pr_number --json body -q .body)
+          echo "::set-output name=pr_number::$pr_number"
+          echo "::set-output name=pr_description::$pr_description"
 
       - name: Extract branch name
         shell: bash
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
+      
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.extract_branch.outputs.branch }}
+      
+      - name: Run make generate
+        run: make generate-with-diff-check
 
       - name: Trigger diff on internal gitlab
+        id: trigger_diff
         run: |
-          curl -X POST \
+          response=$(curl -X POST \
              --fail \
              -F token=${{ secrets.GITLAB_CI_TOKEN }} \
              -F ref=main \
              -F "variables[BRANCH]=${{ steps.extract_branch.outputs.branch }}" \
-             https://git.vshn.net/api/v4/projects/58084/trigger/pipeline
+             https://git.vshn.net/api/v4/projects/58084/trigger/pipeline | jq .web_url)
+          echo "$response"
+          echo "::set-output name=response::$response"
+      
+      - name: Update PR description
+        run: |
+          pr_number=${{ steps.get_pr.outputs.pr_number }}
+          pr_description=${{ steps.get_pr.outputs.pr_description }}
+          new_line="[ ] Checked out Your new diff at: \"${{ steps.trigger_diff.outputs.response }}\""
```

Comment:
Would this add a new line every time we push to the PR?

Maybe it's a bit better structured if we instead add comments?